eng/pipelines/common/templates/runtimes/android-runtime-and-send-to-helix.yml

```diff
@@ -1,83 +1,76 @@
-        coreClrRepoRoot: $(Build.SourcesDirectory)/src/coreclr
-        runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}
-  
-        ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          creator: $(Build.DefinitionName)
-  
-          helixBuild: $(Build.BuildNumber)
-          helixSource: $(_HelixSource)
-  
-          # REVIEW: not sure why "cli" is part of the names here. Leave it for the ones that already had it,
-          # but don't add it to new ones.
-          ${{ if eq(parameters.readyToRun, true) }}:
-            helixType: 'test/functional/r2r/cli/'
-          ${{ if ne(parameters.readyToRun, true) }}:
-            helixType: 'test/functional/cli/'
-  
-          helixQueues: ${{ parameters.helixQueues }}
-  
-          # This tests whether an array is empty
-          ${{ if eq(join('', parameters.helixQueues), '') }}:
-            condition: false
-  
-          publishTestResults: true
-  
-          timeoutPerTestInMinutes: $(timeoutPerTestInMinutes)
-          timeoutPerTestCollectionInMinutes: $(timeoutPerTestCollectionInMinutes)
-  
-          runCrossGen: ${{ and(eq(parameters.readyToRun, true), ne(parameters.crossgen2, true)) }}
-          runCrossGen2: ${{ and(eq(parameters.readyToRun, true), eq(parameters.crossgen2, true)) }}
-          compositeBuildMode: ${{ parameters.compositeBuildMode }}
-          runInUnloadableContext: ${{ parameters.runInUnloadableContext }}
-  
-          ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-            # Access token variable for internal project from the
-            # DotNet-HelixApi-Access variable group
-            helixAccessToken: $(HelixApiAccessToken)
-  
-          helixProjectArguments: '$(Build.SourcesDirectory)/src/coreclr/tests/helixpublishwitharcade.proj'
-  
-          scenarios: normal 
\ No newline at end of file
+steps:
+  - script: $(Build.SourcesDirectory)/src/tests/build$(scriptExt) /p:LibrariesConfiguration=${{ parameters.buildConfig }} -ci -skipstressdependencies -excludemonofailures os Android x64 $(buildConfigUpper)
+    displayName: Build Tests
+
+  #  Send tests to Helix
+  - template: /eng/pipelines/common/templates/runtimes/send-to-helix-step.yml
+    parameters:
+      displayName: Send tests to Helix
+      buildConfig: $(buildConfigUpper)
+      archType: ${{ parameters.archType }}
+      osGroup: ${{ parameters.osGroup }}
+      osSubgroup: ${{ parameters.osSubgroup}}
+      coreClrRepoRoot: $(Build.SourcesDirectory)/src/coreclr
+      runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}
+
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        creator: $(Build.DefinitionName)
+
+        helixBuild: $(Build.BuildNumber)
+        helixSource: $(_HelixSource)
+
+        # REVIEW: not sure why "cli" is part of the names here. Leave it for the ones that already had it,
+        # but don't add it to new ones.
+        ${{ if ne(parameters.readyToRun, true) }}:
```

Comment:
Hmm, I overlooked that one, sorry. You no longer need the condition here. In fact, the corresponding parameters at the top of the script can be removed too (readyToRun, crossgen2, compositeBuildMode, runInUnloadableContext, runtimeFlavor, runtimeFlavorDisplayName).

Reply:
runtimeFlavorDisplayName is still being referenced. I deleted the rest of them.